Makefile

```diff
@@ -18,14 +18,15 @@ heroku:
 
 .PHONY: install
 install:
-	python3 -m venv venv
-	pip install -U pip
 	pip install -r requirements.txt
```

Comment:
Do we need to add `make env-python` above?

Reply:
mmm should be okay I don't think Heroku uses `make install`

Reply:
I had assumed that Heroku used some sort of build pack logic rather than running `make` targets 🥳